src/index.d.ts

```diff
@@ -507,6 +507,24 @@ declare namespace RamdaAdjunct {
          */
         curryRight(fn: Function): Function;
 
+        /**
```

Comment:
Don't forget to sync the description here with the new one in source file that I described.